server/src/main/java/keywhiz/commands/PopulateDevDataCommand.java

```diff
@@ -0,0 +1,139 @@
+        .values(620, 918, 739, OffsetDateTime.parse("2012-06-21T14:38:09.995025Z"), OffsetDateTime.parse("2012-06-21T14:38:09.995025Z"))
+        .values(621, 917, 739, OffsetDateTime.parse("2012-06-21T14:38:09.996522Z"), OffsetDateTime.parse("2012-06-21T14:38:09.996522Z"))
+        .values(622, 918, 740, OffsetDateTime.parse("2012-06-21T14:38:09.998356Z"),
+            OffsetDateTime.parse("2012-06-21T14:38:09.998356Z"))
+        .values(623, 919, 740, OffsetDateTime.parse("2012-06-21T14:38:10.000046Z"), OffsetDateTime.parse("2012-06-21T14:38:10.000046Z"))
+        .values(624, 916, 740, OffsetDateTime.parse("2012-06-21T14:38:10.00146Z"),  OffsetDateTime.parse("2012-06-21T14:38:10.00146Z"))
+        .values(625, 917, 740, OffsetDateTime.parse("2012-06-21T14:38:10.002938Z"), OffsetDateTime.parse("2012-06-21T14:38:10.002938Z"))
+        .values(626, 918, 741, OffsetDateTime.parse("2012-06-21T14:38:11.984113Z"), OffsetDateTime.parse("2012-06-21T14:38:11.984113Z"))
+        .values(627, 917, 741, OffsetDateTime.parse("2012-06-21T14:38:11.990935Z"), OffsetDateTime.parse("2012-06-21T14:38:11.990935Z"))
+        .execute();
+
+    dslContext
+        .insertInto(MEMBERSHIPS, MEMBERSHIPS.ID, MEMBERSHIPS.GROUPID, MEMBERSHIPS.CLIENTID,
+            MEMBERSHIPS.CREATEDAT, MEMBERSHIPS.UPDATEDAT)
+        .values(659, 917, 768, OffsetDateTime.parse("2012-06-21T14:38:09.957063Z"), OffsetDateTime.parse("2012-06-21T14:38:09.957063Z"))
+        .values(660, 918, 769, OffsetDateTime.parse("2012-06-21T14:38:09.970642Z"), OffsetDateTime.parse("2012-06-21T14:38:09.970642Z"))
+        .values(661, 916, 769, OffsetDateTime.parse("2012-06-21T14:38:09.972122Z"), OffsetDateTime.parse("2012-06-21T14:38:09.972122Z"))
+        .values(662, 917, 769, OffsetDateTime.parse("2012-06-21T14:38:09.974132Z"), OffsetDateTime.parse("2012-06-21T14:38:09.974132Z"))
+        .values(663, 919, 770, OffsetDateTime.parse("2012-06-21T14:38:09.975571Z"), OffsetDateTime.parse("2012-06-21T14:38:09.975571Z"))
+        .values(664, 917, 770, OffsetDateTime.parse("2012-06-21T14:38:09.976875Z"), OffsetDateTime.parse("2012-06-21T14:38:09.976875Z"))
+        .values(665, 918, 771, OffsetDateTime.parse("2012-06-21T14:38:09.978106Z"), OffsetDateTime.parse("2012-06-21T14:38:09.978106Z"))
+        .values(666, 919, 771, OffsetDateTime.parse("2012-06-21T14:38:09.979935Z"), OffsetDateTime.parse("2012-06-21T14:38:09.979935Z"))
+        .values(667, 918, 772, OffsetDateTime.parse("2012-06-21T14:38:09.981239Z"), OffsetDateTime.parse("2012-06-21T14:38:09.981239Z"))
+        .values(668, 917, 772, OffsetDateTime.parse("2012-06-21T14:38:09.982586Z"),
+            OffsetDateTime.parse("2012-06-21T14:38:09.982586Z"))
+        .execute();
+
+    dslContext.insertInto(USERS)
+        .set(USERS.USERNAME, "keywhizAdmin")
+        .set(USERS.PASSWORD_HASH, BCrypt.hashpw("adminPass", BCrypt.gensalt()))
```

Comment:
If you want, it could be really nice to make a constant BasicCredentials login value. The value could be statically imported here and also in tests.